### ISSUE TITLE
function_call_space removes space also from function definitions

### ISSUE THREAD

**Issue by JanisE** [FriendsOfPHP/PHP-CS-Fixer#1498]
> function_call_space When making a method or function call, there MUST NOT be a space between the method or function name and the opening parenthesis.

This fixer removes also space after function name in a function definition, not just a call.

This fixer needs a rename in any case. Although I would love to see a fixer that removes space only from function calls and leaves it in definitions.

**Comment by keradus** [comment]
> This fixer removes also space after function name in a function definition, not just a call.

Could you point to it?

**Comment by JanisE** [comment]
Erm... I took this from the documentation. Do you mean link to this file? https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Fixer/PSR2/FunctionCallSpaceFixer.php

**Comment by keradus** [comment]
> This fixer removes also space after function name in a function definition, not just a call.

Please point to that part directly. Which line?

**Comment by JanisE** [comment]
Oh, you mean, you cannot reproduce it. Sorry!

test.php:
```
<?php

function TisMy ($p1)
{
	print $p1;
}
```
Gets formatted to:
```
<?php

function TisMy($p1)
{
	print $p1;
}
```
By:
```
./php-cs-fixer.phar fix test.php --config-file test.php_cs
Loaded config from "test.php_cs"
   1) /home/janis/bin/phpcsfixer/test.php
Fixed all files in 0.067 seconds, 5.750 MB memory used
```
Where test.php_cs:
```
<?php

$finder = Symfony\CS\Finder\DefaultFinder::create()
	->in(__DIR__)
;

return Symfony\CS\Config\Config::create()
	->level(Symfony\CS\FixerInterface::NONE_LEVEL)
	->fixers([
		'function_call_space',
	])
	->finder($finder)
;
```

**Comment by keradus** [comment]
Oh, that's what you mean!

You have quoted that line like it was description of fixer

**Comment by keradus** [comment]
```
> sdf
asdfafds
asdfasfd
sadfdafs
```

is all the one quote:
> sdf
asdfafds
asdfasfd
sadfdafs

if you want to quote only one line you must put empty one after it, like:
````
> sdf

asdfafds
asdfasfd
sadfdafs
```

> sdf

asdfafds
asdfasfd
sadfdafs

**Comment by JanisE** [comment]
Done.

**Comment by SpacePossum** [comment]
ping @keradus this one can be closed :)

**Comment by keradus** [comment]
Thanks ;)
If the content of first post will be sth like "Closes #1498" then the issue will be automatically closed during merging.

**Comment by SpacePossum** [comment]
Cool, didn't know, will keep in mind for next time :)
